sql/handler.cc

```diff
@@ -1424,7 +1424,7 @@ int ha_commit_trans(THD *thd, bool all)
     if (is_real_trans)
       thd->transaction.cleanup();
 #ifdef WITH_WSREP
-    if (WSREP(thd) && all && !error)
+    if (all && !error)
     {
```

Comment:
why removing WSREP(thd) here? it looks like there will be problems for non wsrep threads within wsrep_commit_empty()

Reply:
The decision to enter wsrep commit hooks is done internally in `wsrep_commit_empty()` etc. But I guess you are right, it might be better to make it explicitly visible in handler code that the thd is indeed processing a wsrep transaction. Will update the PR.